pkg/fs/xfs/xfs.go

```diff
@@ -18,20 +18,25 @@ package xfs
 
 import (
 	"encoding/binary"
-	"encoding/hex"
-	"github.com/google/uuid"
-	"os"
-)
+	"fmt"
+	"io"
 
-type XFS struct {
-	SuperBlock *XFSSuperBlock
-}
+	fserrors "github.com/minio/direct-csi/pkg/fs/errors"
+)
 
-func NewXFS() *XFS {
-	return &XFS{}
+// UUID2String converts UUID to string.
+func UUID2String(uuid [16]byte) string {
```

Comment:
Can we move this function to a common place and use it in ext4.go also?

Reply:
Each filesystem represents UUID data in a different format. It is OK to be duplicated this. Why this function exported is someone can directly use `xfs` or any filesystem code at low level and calls this function for UUID to string conversion if needed.